projects/taskntalk/coopgame_agent.py

```diff
@@ -25,14 +25,14 @@ class CooperativeGameAgent(Agent):
     """
 
     OPTIM_OPTS = {
-        'adadelta': optim.Adadelta,
-        'adagrad': optim.Adagrad,
+        'adadelta': optim.Adadelta,  # type: ignore
```

Comment:
this is everywhere lol

Reply:
yes recall when we used to copy pasta all the agents?

Reply:
🤦‍♂